streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -0,0 +1,132 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.openmrs.analytics;
+
+import io.debezium.data.Envelope;
+import org.apache.camel.*;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.debezium.DebeziumConstants;
+import org.apache.camel.model.dataformat.JsonLibrary;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+
+import static org.openmrs.analytics.PipelineConfig.getDebeziumConfig;
+
+/**
+ * Debezium change data capture / Listener
+ */
+public class DebeziumListener  extends RouteBuilder {
+
+    private static final Logger LOG = LoggerFactory.getLogger(DebeziumListener.class);
+    private static final String FHIR_ROUTE_HANDLER = "direct:get-fhir";
+
+    // FhirStore
+    private static final FhirStoreUtil FHIR_STORE_UTIL  = new FhirStoreUtil(System.getProperty("cloud.gcpFhirStore","gcp"));
```

Comment:
Yes, Integrates well with docker container and IDE. Also, system properties come in handy when you have multiple parameters to pass. In this case, we have over 15 parameters to pass and could be messy passing them as command flags.

Reply:
We don't need to address this issue in this PR, so I am just putting it here since we have a context to discuss it: While I understand it is a subjective choice I prefer using command line args and then environment variables. Benefits:
1) Usually there are libraries to easily handle default values and help messages nicely.
2) If there is a typo in an argument, the command line parse catches it but with env var. or properties with defaults, it won't.
3) If we prefer non-command line args approach, I think env. variables are better since they have more direct integration with non-java tools/langs, e.g., bash, or docker, or other languages (for example it is possible that we may have some Python in the future).

But again, let's not address it in this PR and discuss it further at some point (please feel free to reply with what you think).

Reply:
I see what you mean, It can will be important to provide users with a validation mechanism of the input args! I am going ahead to create a ticket/issue to standardize this across all the 4 subprojects. Thanks @bashir2

Reply:
I have created an issue here: https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/issues/21

Reply:
Thanks.